src/Pos/Wallet/Web/Api.hs

```diff
@@ -6,18 +6,33 @@
 module Pos.Wallet.Web.Api
        ( WalletApi
        , walletApi
+       , WalletApiWithDocs
+       , walletApiWithDocs
+       , swaggerSpecForWalletApi
        ) where
 
 import           Data.Proxy                 (Proxy (Proxy))
 
-import           Pos.Types                  (Coin, SoftwareVersion)
-import           Pos.Wallet.Web.ClientTypes (CAddress, CCurrency, CInitialized, CProfile,
-                                             CTx, CTxId, CTxMeta, CUpdateInfo, CWallet,
-                                             CWalletInit, CWalletMeta, CWalletRedeem,
-                                             SyncProgress)
+import           Pos.Types                  (Coin, SoftwareVersion, ApplicationName,
```

Comment:
Imports grouping still doesn't comply with the [guide](https://github.com/serokell/serokell-util/blob/master/serokell-style.md#imports).